fluentd/Dockerfile.centos7

```diff
@@ -1,35 +1,39 @@
-FROM centos/ruby-24-centos7:latest
+FROM centos:centos7
 
 MAINTAINER OpenShift Development <dev@lists.openshift.redhat.com>
 
 ENV DATA_VERSION=1.6.0 \
-    FLUENTD_VERSION=0.12.41 \
-    FLUENTD_AUDIT_LOG_PARSER_VERSION=0.0.5 \
```

Comment:
Why do we also drop the `FLUENTD_AUDIT_LOG_PARSER_VERSION` variable?

Reply:
It is now unused